subsys/bluetooth/host/mesh/lpn.c

```diff
@@ -213,7 +213,7 @@ static void clear_friendship(bool force, bool disable)
 
 	k_delayed_work_cancel(&lpn->timer);
 
-	bt_mesh_friend_cred_del(bt_mesh.sub[0].net_idx, lpn->frnd);
+	friend_cred_del(bt_mesh.sub[0].net_idx, lpn->frnd);
```

Comment:
Not currently, since Zephyr only supports being an LPN on the first subnet. Once we add support for being an LPN on multiple subnets, then the clearing will be for the subnet of that particular Friendship (there will be two Friendship contexts if a Friend-LPN pair does Friendship on two subnets).